### DISCUSSION TITLE
Multiple Windows Images concurrently

### DISCUSSION THREAD

**Discussion by braninl** [vmware/packer-examples-for-vsphere#943] [Help] [answered]
With the old version of build.sh, I was able to build all related Windows images concurrently (e.g. Windows Server 2022 Standard core, Windows Server 2022 Standard DEXP, Windows Server 2022 Datacenter Core, Windows Server 2022 Datacenter DEXP).  It seems like this has gone away with the current project.json-based build method.  Is there still a way to build multiple images concurrently (including ones like Windows 11 Professional and Windows 11 Enterprise), or do they all need to be selected individually now (which vastly increases the time it takes to build all the Windows images)?

Thanks.

Branin

**Comment by tenthirtyam** [comment, marked as answer]
Yes! Simply run the following:

```shell
packer init builds/windows/server/2022/.

packer build -force on-error=ask \
    -var-file="config/build.pkrvars.hcl" \
    -var-file="config/common.pkrvars.hcl" \
    -var-file="config/vsphere.pkrvars.hcl" \
    builds/windows/server/2022
```
